app/src/screens/about/index.tsx

```diff
@@ -8,8 +8,11 @@ import { appVersion } from 'utils/electron';
 import { autoUpdateRequest } from 'utils/electron';
 import AutoUpdateStatus from './AutoUpdateStatus';
 import TextButton from './TextButton';
+import useShortcuts from 'hooks/useShortcuts';
+
+export default function About() {
+  useShortcuts()
```

Comment:
`useShortcuts` is used now for every screen. Best to be in [App](https://github.com/RStankov/FocusedTask/blob/1ad9a378b9792f5340eba6b1f51d12b5f0c36efc/app/src/screens/index.tsx) component. In this way it will be initialized once